sklearn/inspection/_plot/tests/test_plot_partial_dependence.py

```diff
@@ -93,6 +93,23 @@ def test_plot_partial_dependence(grid_resolution, pyplot, clf_boston, boston):
     assert ax.get_ylabel() == boston.feature_names[1]
 
 
+@pytest.mark.parametrize("individual, shape", [
+    (False, (1, 3)), (True, (1, 3, 506)), ('both', (1, 3, 507))
+])
+def test_plot_partial_dependence_individual(individual, shape,
```

Comment:
The error about matplotlib is raised because you need to add `pyplot` fixture which will skip the test when required

```suggestion
def test_plot_partial_dependence_individual(pyplot, individual, shape,
```